### DISCUSSION TITLE
The Raven Dynasty

### DISCUSSION THREAD

**Discussion by nvchks** [Xinboshin/TheRaven-Dynasty#14] [Уведомления]
<hr /><em>This discussion was created from the release <a href='https://github.com/Xinboshin/TheRaven-Dynasty/releases/tag/v1.0.1-dev'>The Raven Dynasty</a>.</em>

**Comment by nvchks** [comment]
Данные дискуссии требуются для предложений и пожеланий по моду.

**Comment by nik1000-7** [comment]
1

**Reply by nvchks** [reply]
Великодушный, здравствуйте. Вы надоели присылать бесполезные комментарии и коммиты в сторону дискуссий данного репозитория. Пожалуйста, предоставьте что-либо стоящее для модификации, либо, более не участвуйте в обсуждениях.
